app/furniture/marketplace/checkout.rb

```diff
@@ -47,7 +47,14 @@ def stripe_line_items
           quantity: cart_product.quantity,
           adjustable_quantity: {enabled: true}
         }
-      end
+      end + [
+        {quantity: 1,
+         price_data: {
+           currency: "USD",
+           unit_amount: marketplace.delivery_fee_cents,
+           product_data: {name: "Delivery"}
+         }}
+      ]
```

Comment:
I wonder if this would be a bit more readable with this one element first and then ` + <result of map` ?

Reply:
Good call, I think splitting it into two separate methods and joining them explicitely is even clearer!